openreview/conference/helpers.py

```diff
@@ -64,6 +65,11 @@ def get_conference(client, request_form_id):
 
     builder.set_override_homepage(True)
     conference = builder.get_result()
-    conference.open_submissions(start_date = submission_start_date, due_date = submission_due_date)
+
+    submission_additional_options = note.content.get('Additional Submission Options', {})
+    if submission_additional_options and isinstance(submission_additional_options, str):
```

Comment:
you don't need to check string instance, the note.content.get can return an None if there is no additional options.

Reply:
Removed the check for data type of _note.content.['Additional Submission Options']_ as it is always a string.
As discussed, we can not pass None in additional options due to the way we are processing additional options as of now. I will change that in a separate PR.